lang/po/Firmware_sv.po

```diff
@@ -2626,7 +2626,7 @@ msgstr "Ett oväntat fel inträffade."
 #~ msgstr "TM autoinst fallerad"
 
 #~ msgid "Temp. model autotune"
-#~ msgstr "Temp modell autoinst"
+#~ msgstr "Thermal Modell autoinst"
```

Comment:
"Temp. model" messages no longer exist, so this is kind of pointless. I would drop this, and in a new commit purge the obsolete translations.

Reply:
Done